### PR TITLE
Tiny update of migrating index

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: source/migrating/index.rst [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
+GMT 在大版本升级时（如 GMT 5 升级为 GMT 6），语法及用法上会发生一些变化。
 但由于 GMT 会尽可能兼容旧版本的语法，用户往往只需要对旧版本 GMT 语法编写的脚本做少量修改，
 即可在新版本 GMT 中运行。
 
 这一章将介绍如何实现：
 
-#. 将GMT4脚本迁移到GMT5
-#. 将GMT5脚本迁移到GMT6
-#. 将经典模式语法迁移到GMT6现代模式语法
+#. 将 GMT4 脚本迁移到 GMT5：\ doc:`/migrating/4to5`
```
Comment (seisman) [inline source/migrating/index.rst:15]: 可以把这个列表与 `toctree` 合并

Comment (core-man) [reply source/migrating/index.rst]: See eedb948?

File: source/migrating/index.rst [hunk 2]
```diff
@@ -3,23 +3,35 @@
...
+
    5to6
+
+将经典模式语法迁移到 GMT6 现代模式语法：
+
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+
    classic2modern
```
Comment (seisman) [inline source/migrating/index.rst:37]:
```
.. toctree::
   :maxdepth: 1

   将GMT4脚本迁移到GMT5 <4to5>
   将GMT5脚本迁移到GMT6 <5to6>
   将经典模式语法迁移到GMT6现代模式语法 <classic2modern>
```

Comment (core-man) [reply source/migrating/index.rst]: Fixed in 84db026

**Comment by seisman** [comment]
Not good. It also affects the sidebar:

image

Perhaps reverting to your initial commit 9ad56bff23b14641a69bf0c169318faf48ac3466 is better?

File: source/migrating/index.rst [hunk 3]
```diff
@@ -3,18 +3,18 @@
...
 即可在新版本 GMT 中运行。
 
 这一章将介绍如何实现：
 
-#. 将GMT4脚本迁移到GMT5
-#. 将GMT5脚本迁移到GMT6
-#. 将经典模式语法迁移到GMT6现代模式语法
+#. 将 GMT4 脚本迁移到 GMT5：\ doc:`/migrating/4to5`
+#. 将 GMT5 脚本迁移到 GMT6：\ doc:`/migrating/5to6`
+#. 将经典模式语法迁移到 GMT6 现代模式语法：\ doc:`/migrating/classic2modern`
```
Comment (seisman) [inline source/migrating/index.rst:17]:
```suggestion
#. 将 GMT4 脚本迁移到 GMT5：\ :doc:`/migrating/4to5`
#. 将 GMT5 脚本迁移到 GMT6：\ :doc:`/migrating/5to6`
#. 将经典模式语法迁移到 GMT6 现代模式语法：\ :doc:`/migrating/classic2modern`
```

**Review by seisman — APPROVED** [review]

**Pull request merged**